packit/upstream.py

```diff
@@ -244,10 +244,31 @@ def get_specfile_version(self) -> str:
         logger.info(f"Version in spec file is {version!r}.")
         return version
 
+    @property
+    def get_current_version_env(self):
+        for packit_package_name, package_config in self.package_config.packages.items():
+            packit_upstream_package_name = (
+                package_config.upstream_package_name
+                if package_config.upstream_package_name
+                else ""
+            )
+            packit_downstream_package_name = (
+                package_config.downstream_package_name
+                if package_config.downstream_package_name
+                else ""
+            )
+            env = {}
+            env["PACKIT_PACKAGE_NAME"] = packit_package_name
+            env["PACKIT_UPSTREAM_PACKAGE_NAME"] = packit_upstream_package_name
+            env["PACKIT_DOWNSTREAM_PACKAGE_NAME"] = packit_downstream_package_name
+
+            # there should be always only one package associated with
```

Comment:
Do we need a for-loop if there's always just one package? :thinking: sorry, I got a bit confused when I saw the loop and then even more when I noticed the `return` in the first iteration :D

Reply:
I know this is misleading, and it comes from [here]( https://github.com/packit/packit/blob/73d04ae8dbf623b9e2d9f0acb03668173e8c966d/packit/config/package_config.py#L250-L253) until we do not refactorize `PackageConfig` and `PackageConfigView` I think there is no better way. Or at least I don't see it. I can make a check to be sure there is just one entry and log an error otherwise. I did not do that, because I think it should never happen.